_about/ROADMAP.md

```diff
@@ -0,0 +1,64 @@
+# Q3 2020 Roadmap
+
+Each quarter the team will highlight areas of focus for our work and upcoming research.
+ 
+We select items to include in the roadmap from community issues and internal priorities. When community pull requests exist for an item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp may complete any additional work needed. 
+
+Each release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. 
+
+To make contribution easier, we’ll be using the [`Help Wanted`](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) tag to point to issues we’d like to include in this quarter’s series of releases. 
+
+This quarter (July-September ‘20) we will prioritze the following areas of work: 
+
+## Currently In Progress
+
+### 2.0.0 
+
+Major version releases include code removals, deprecations, and breaking changes. A corresponding “upgrade guide” will be published alongside the release. 
+
+In the 2.0.0 release we'll focus on three areas:
+ - Upgrading to [Terraform Plugin SDK v2](https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html)
+ - [Removing support for Terraform 0.11](https://github.com/hashicorp/terraform-provider-kubernetes/issues/905)
+ - [Changing provider credentialling]
+
+### Coverage
+
+Issues: [theme/coverage](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Atheme%2Fcoverage)
+
+Goal: Improve coverage of the Kubernetes API
+
+While our goal for the Kubernetes provider is complete coverage of the Kubernetes API, this is more work than we can accomplish in one quarter.  Initially, we're prioritizing this work based on user feedback, but we'll also look at ways to address coverage gaps more proactively.
+
+### Lifecycle
+
+Issues: [theme/lifecycle](https://github.com/hashicorp/terraform-provider-kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Atheme%2Flifecycle)
+
+Goal: Lifecycle improvements to match user workflows
+
+There are several requests to modify behavior based on Kubernetes lifecycle events, for example, waiting or not waiting for a pod to become ready before a deployment is considered complete. We’ll look at this and other areas where additional lifecycle controls make sense in the provider. 
+
+### Errors
```

Comment:
What about designating this as a stretch goal given we have 10 weeks left in the quarter? There's going to be some collateral unnamed work around 0.13 compatibility, SDK upgrades, theme/diffs and the test infrastructure improvements we're talking about. Happy to discuss offline to decide on this.

Reply:
Sounds reasonable